src/components/ChatEntry.js

```diff
@@ -1,22 +1,44 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
-  return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
-      <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
-      </section>
-    </div>
-  );
+    const likeButton = () => {
+        const updatedChatEntry = {
+            id: props.id,
+            sender: props.sender,
+            body: props.body,
+            timeStamp: props.timeStamp,
+            liked: !props.liked,
+        };
+        props.onUpdateChatEntry(updatedChatEntry);
+    };
+
+    const heart = props.liked ? '❤️' : '🤍';
```

Comment:
Nice job using the liked value coming from props to update the component display. There's no need to track this with local state, since any time we update the state (using the supplied callback) the component data will update, which we will receive through updated props.